sct.py

```diff
@@ -168,6 +169,14 @@ def convert(self, value, param, ctx):
         return value
 
 
+class SctLoader(unittest.TestLoader):
+    def getTestCaseNames(self, testCaseClass):
+        test_cases = super().getTestCaseNames(testCaseClass)
+        assert len(
+            test_cases) < 2, f"SCT expect only one test case to be selected, found {len(test_cases)}:\n{pprint.pformat(test_cases)}"
```

Comment:
Technically, the length of the `test_cases` collection object gets calculated here twice. Always.
Please, calculate it before the assertion and then just use the new var.

Reply:
fixed